wikicurses/main.py

```diff
@@ -35,6 +37,111 @@ def tabComplete(text, matches):
     return match
 
 
+def showImages():
+    """
+    Launch program to display the images for the current article.
+
+    The program is specified in the config, along with it's args.
+    Otherwise, the default is `feh`.
+    """
+    if 'wikipedia.org' not in wiki.siteurl:
+        ex.notify('Cancelled: Only works on Wikipedia')
+        return
```

Comment:
I wasn't sure how standardised any of this was to be honest. When I tried it on a couple of others, they raised errors (I believe to do with a different JSON structure.) I didn't want to risk crashing the user's session. However if you would prefer I'll remove that limit, and put the call to the `showImages` function in a try statement?